test/filter-test.js

```diff
@@ -334,4 +335,26 @@ describe('Filter', function() {
     );
     assert.bufferEqual(actual, expected);
   });
+
+  describe('HTTP', function() {
+    const nclient = new NodeClient({
+      port: node.network.rpcPort
+    });
+
+    it('should get info', async () => {
+      const info = await nclient.getInfo();
+      const {indexes} = info;
+      const {filter} = indexes;
```

Comment:
```suggestion
      const {indexes: {filter}} = await nclient.getInfo();
```

maybe do this instead? just because it's cooler and neater 🚀

Reply:
thanks! fixed in dabdd7bf83bd27576e5d25c1184d962a4f7e1da2